src/admin-views/troubleshooting/ea-status.php

```diff
@@ -17,28 +17,30 @@
 	<?php esc_html_e( 'Event Aggregator system status ', 'tribe-common' ); ?>
 </h3>
 
-<table class="tribe-events-admin__ea-status-table">
-	<?php
-		// license key
-		include_once Tribe__Main::instance()->plugin_path . 'src/admin-views/troubleshooting/ea-status/license-key.php';
-			// if EA is not active, bail out of the rest of this
-			if ( ! $ea_active ) {
-				echo '</table>';
-				return;
-			} else {
-				// current usage
-				include_once Tribe__Main::instance()->plugin_path . 'src/admin-views/troubleshooting/ea-status/current-usage.php';
-				// server connection
-				include_once Tribe__Main::instance()->plugin_path . 'src/admin-views/troubleshooting/ea-status/server-connection.php';
-				// scheduler status
-				include_once Tribe__Main::instance()->plugin_path . 'src/admin-views/troubleshooting/ea-status/scheduler-status.php';
+<div class="tribe-events-admin__ea-status-table-wrapper">
+	<table class="tribe-events-admin__ea-status-table">
+		<?php
+			// license key
+			include_once Tribe__Main::instance()->plugin_path . 'src/admin-views/troubleshooting/ea-status/license-key.php';
+				// if EA is not active, bail out of the rest of this
+				if ( ! $ea_active ) {
+					echo '</table></div>';
+					return;
+				} else {
```

Comment:
spacing. also, this case, it would be better if we just checked if ea is active, rather than echoing the closing elements if not active.

```
if ( $ea_active ) {
	// do all includes
}
```

Having 2 places for closing tags makes this brittle and requires us to change multiple places rather than one place

Reply:
@paulmskim yeah that makes sense, I've updated the conditional statement. Working on the artifacts now.